atomate2/openmm/flows/core.py

```diff
@@ -0,0 +1,51 @@
+
+from atomate2.openmm.jobs.core import InputMaker, EnergyMinimizationMaker, NPTMaker, AnnealMaker, NVTMaker
+from typing import List, Union, Optional
+from pathlib import Path
+
+from dataclasses import dataclass
+
+from pydantic import Field
+from jobflow import Maker, Flow
+
+from pymatgen.io.openmm.schema import InputMoleculeSpec
+
+
+
+@dataclass
+class ProductionMaker(Maker):
+    """
+
+
+    """
+    name: str = "production"
+
+    input_maker: InputMaker = Field(default_factory=InputMaker)
+    energy_maker: EnergyMinimizationMaker = Field(default_factory=EnergyMinimizationMaker)
+    npt_maker: NPTMaker = Field(default_factory=NPTMaker)
+    anneal_maker: AnnealMaker = Field(default_factory=AnnealMaker)
+    nvt_maker: NVTMaker = Field(default_factory=NVTMaker)
```

Comment:
To make sure I'm seeing the bigger picture, is this where we could add a Job for re-parameterization (e.g. using Bespokefit) or would it be its own flow?

Reply:
That'd probably be it's own flow, but this is exactly where we could add a job.